firmware/controllers/sensors/sensor.cpp

```diff
@@ -12,6 +14,30 @@ struct SensorRegistryEntry {
 
 static SensorRegistryEntry s_sensorRegistry[static_cast<size_t>(SensorType::PlaceholderLast)] = {};
 
+static const char* s_sensorNames[] = {
+	"Invalid",
```

Comment:
dirty magic index matching? kind of hacky and poorly documentad coupling between enum and this array?

Reply:
that's what the static assert is for - it at least guarantees that the sizes match

Reply:
that's not enough. this has to designed better or documented better IMHO